android-vjestina-homework-movie-app-data-mock/app/src/main/java/agency/five/codebase/android/movieapp/ui/moviedetails/MovieDetailsScreen.kt

```diff
@@ -0,0 +1,283 @@
+
+        Column(
+            modifier = Modifier
+                .fillMaxSize(),
+            verticalArrangement = Arrangement.Bottom,
+        ) {
+            Row(
+                modifier = Modifier
+                    .fillMaxWidth()
+                    .padding(start = MaterialTheme.spacing.medium),
+                verticalAlignment = Alignment.CenterVertically
+            ) {
+                UserScoreProgressBar(
+                    score = movieDetailsViewState.voteAverage
+                )
+
+                Text(
+                    text = stringResource(id = R.string.user_score),
+                    modifier = Modifier
+                        .padding(start = MaterialTheme.spacing.small),
+                    fontSize = 16.sp,
+                    fontWeight = FontWeight.Bold,
+                    color = Color.White
+                )
+            }
+
+            Spacer(
+                modifier = Modifier
+                    .height(MaterialTheme.spacing.medium)
+            )
+
+            Text(
+                text = movieDetailsViewState.title,
+                modifier = Modifier
+                    .padding(start = MaterialTheme.spacing.medium),
+                fontSize = 20.sp,
+                fontWeight = FontWeight.Bold,
+                color = Color.White
+            )
+
+            Spacer(
+                modifier = Modifier
+                    .height(MaterialTheme.spacing.medium)
+            )
+
+            FavoriteButton(
+                isFavourite = movieDetailsViewState.isFavorite,
+                modifier = Modifier
+                    .padding(
+                        start = MaterialTheme.spacing.medium,
+                        bottom = MaterialTheme.spacing.medium
+                    )
+                    .size(30.dp),
+                onClick = { }
+            )
+        }
+    }
+}
+
+@Composable
+fun MovieDetailsOverview(
+    modifier: Modifier = Modifier,
+    movieDetailsViewState: MovieDetailsViewState
+) {
+    Column(
+        modifier = modifier
+            .fillMaxWidth()
+            .padding(start = MaterialTheme.spacing.medium, end = MaterialTheme.spacing.medium)
+    ) {
+        Text(
+            text = stringResource(id = R.string.overview),
+            maxLines = 1,
+            modifier = Modifier
+                .fillMaxWidth(),
+            fontSize = 18.sp,
+            fontWeight = FontWeight.Bold
+        )
+
+        Spacer(
+            modifier = Modifier
+                .height(MaterialTheme.spacing.small)
+        )
+
+        Text(
+            text = movieDetailsViewState.overview,
+            maxLines = 7,
+            modifier = Modifier
+                .fillMaxWidth(),
+            fontSize = 14.sp
+        )
+    }
+}
+
+@Composable
+fun MovieDetailsCrew(
+    modifier: Modifier = Modifier,
+    movieDetailsViewState: MovieDetailsViewState
+) {
+    Column(
+        modifier = modifier
+            .heightIn(min = 50.dp, max = 400.dp)
```

Comment:
Why do you need this?

Reply:
Without it the app crashes when I click on movie to show movie details screen, run log: Vertically scrollable component was measured with an infinity maximum height constraints, which is disallowed.

Reply:
@ivanseki Please address my comment above before merging.

Reply:
Ok I understand, but then do I need to assume that there will always be fixed amount of crew?

Reply:
Asked Matej and he said that I should use first 6 from list even if list is greater. Decided to make new composable item called CrewColumn and then used it in Crew part giving it fixed indexes from list as arguments.

Reply:
@ivan-varga, answered but forgot to tag you